framework/templates/ecl-filters/README.md

```diff
@@ -0,0 +1,11 @@
+# Filters
+
+## When and how to use
+
+When a list or collection of content items with shared characteristics is too large, filters allow users to narrow the scope of the displayed items.
+
+- the filter has a H3 title and consists of input fields (with labels) and a 'clear all' action button
+- all filter only allow for single select. This means that per search, only 1 value can be selected
+- the first filter should always be a search field labeled Keywords
+- always use a vertical, 1 column layout
+- selecting a value in an input field automatically refreshes the list of displayed items
```

Comment:
The thing is, we don't have the list of results yet. Would be worth upgrading this component once we have them. 

Regarding the doc, I stopped at "DO NOT COPY THIS INFO ON THE ECL"